src/components/HomeSectionClients/styled.ts

```diff
@@ -71,6 +85,19 @@ const Logo = styled.img`
   `};
 `;
 
+export const TagContainer = styled.div`
+  margin-top: 4px;
+
+  div {
+    display: inline-block;
+    background: ${colors.brightYellow};
+    margin-top: 4px;
```

Comment:
I see what you’ve done here! Thanks for adding styles for tags that are wrapping on a different line.

May I recommend a different approach? What I like to use nowadays when I’m styling wrapping elements is a combination of `display: flex`, `flex-wrap: wrap`, and `gap`. For example:

```css
.tagContainer {
  display: flex;

  /* Make it wrap, as if all children were inline-block */
  flex-wrap: wrap;

  /* Add an 8px gap between every child element. `gap: 4px 8px` also works
     (for different vertical and horizontal spacing.) */
  gap: 8px;
}

.tag {
  /* Prevent elements from widening or narrowing. */
  flex: none;
}
```

What I like about this approach is that it’s way more maintanable. Instead of maintaining setting `margin-top: 4px` on a `.tag` and then remembering to decrease a `margin-top` on the `.tagContainer`, you just change a single `gap` property in a single place.

It also reads simpler: the purpose of `gap` is obvious (it’s in the name), whereas multiple `margin`s can confuse another developer.